conda-forge.yml

```diff
@@ -1,6 +1,8 @@
+bot:
+  inspection: update-grayskull
```

Comment:
@jaimergp, I think it'll be useful to have update-grayskull on.  I assume you're good with that.